cse-cc-import-runner-app/src/main/resources/application.yml

```diff
@@ -10,10 +12,28 @@ spring:
       reply-timeout: 2400000
   cloud:
     stream:
+      function:
+        definition: interruptionConsumer;requestProcessor;notification-producer
```

Comment:
Maybe name notificationProducer to keep same syntax

Reply:
use of notificationProducer